### ISSUE TITLE
Prevent CompatHelper PRs from polluting TagBot entries?

### ISSUE THREAD

**Issue by tlnagy** [bcbi/CompatHelper.jl#141]
Is it possible to automatically label CompatHelper PRs with the `no changelog` label so that they don't fill up the TagBot changelog output?

**Comment by DilumAluthge** [comment]
Currently we don’t have that ability. But it would be a cool feature to add. I’d be happy to review a PR that adds this feature to CompatHelper.

**Comment by tlnagy** [comment]
https://github.com/JuliaWeb/GitHub.jl/pull/115 might add the option for labeling new PRs.

**Comment by fredrikekre** [comment]
No, that is just for obtaining labels. You can already add labels, Registrator does it.

**Comment by DilumAluthge** [comment]
Yeah, GitHub.jl already has the ability to add labels to a PR. Someone just has to add the feature to CompatHelper.

I think it should be an opt-in feature. So the default behavior is that no labels are added. And then you can opt in by providing a list of one or more labels that you would like to add.
